libs/frontend/feature-build/src/lib/dnd/builder-dnd-component.ts

```diff
@@ -94,7 +94,9 @@ const renderImageHover = (
     {
       'data-component-id': component.id,
       style: {
-        display: 'flex',
+        // Use inline-block to simulate the bottom space caused by the inline behavior of <img>.
+        // See https://stackoverflow.com/a/5804278/19772349 for more information.
+        display: 'inline-block',
```

Comment:
I noticed that there's still a strange jump issue even though the `Math.floor` when calculating the dimensions of the image hover wrap has been removed.

https://github.com/samatechtw/pubstudio-builder/assets/59676941/c40a9e57-555e-4c8b-977c-ed0d89721e1f

After some investigation I learned that the `<img>` in HTML is in inline display, so it actually "preserves" some spaces at the bottom for some characters. After changing the `display` of the hover wrap from `flex` to `inline-block`, the problem seems to be solved!

Reply:
Good catch!